proctor-common/src/main/java/com/indeed/proctor/common/ProctorResult.java

```diff
@@ -193,4 +197,8 @@ public Identifiers getIdentifiers() {
     public Map<String, Object> getInputContext() {
         return inputContext;
     }
+
+    public boolean markTestAsLogged(String test) {
```

Comment:
NIT: parameter should be testVersion

Reply:
Same here, I think testVersion is confusing. We are using "version" to respresent a test.